README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED JANUARY 2022 IN FAVOR OF https://app.datadoghq.com/ci/pipelines
```

Comment:
@brianshef Are there steps to setup the new datadog ci pipelines for each repo?

Reply:
Nope, the integration is automatic with all the repos in our org using GitHub Actions.